schemas/metaschema/domains.schema

```diff
@@ -66,4 +67,19 @@
   Volume: { type: 'number', unit: 'Litre', symbol: 'l' },
   Speed: { type: 'number', unit: 'Metres per second', symbol: 'm/s' },
   Ratio: { type: 'number' },
+  ActionType: Enum(
+    'Create', 'Update', 'Delete', 'ShowDetail',
+    'Digicert', 'Steps', 'Custom'
```

Comment:
What's the `Steps` action type, how will it behave?

Reply:
As of now it's not really specified how will `steps` work.
It describes an Action that may happen via multiple interactions with the user (and corresponding executions of different Actions) while keeping some internal state.

There is no use for it now so I can remove it, though we will probably need to add it later anyway.

Reply:
Doesn't this PR add an `Execute` decorator, that is supposed to determine the next step in the interaction with the user?
Or did I misunderstand the purpose of it and there is a significant difference?

Reply:
Yes, this is a basic idea - to return the next step via Execute, but that may not be enough when we actually get to implementing such an interaction.

Reply:
`'Create', 'Update', 'Delete'` is ok, but we don't need `'ShowDetail', 'Digicert', 'Steps', 'Custom'` @lundibundi